src/transformState.js

```diff
@@ -61,8 +61,27 @@
  * @param {Object} state
  * @param {Object[]} transforms
  */
-function transformState(state, transforms) {
-  // write code here
+
+function transformState(state, transform) {
+  transform.map(item => {
```

Comment:
Збережи проміжний проміжний результат `map` метода і винеси в змінну
```suggestion
  transforms.map(transform => {
```